docker-compose.override.yml

```diff
@@ -24,7 +24,7 @@ services:
     environment:
       - MINIO_ACCESS_KEY=minioadmin
       - MINIO_SECRET_KEY=minioadmin
-      - MINIO_ADDRESS=http://localhost:9000
+      - MINIO_ADDRESS=http://minio:9000
```

Comment:
It might be worth adding a line in the main README (around [here](https://github.com/CMSgov/easi-app#setup-minio)) telling people about the /etc/hosts change.